tdrs-backend/tdpservice/conftest.py

```diff
@@ -112,41 +113,76 @@ def infected_file():
     )
 
 
-def create_temporary_file(
-    file_contents: str,
-    suffix: str = '.txt'
-) -> NamedTemporaryFile:
+def create_temporary_file(file, file_name) -> SimpleUploadedFile:
     """Create a temporary file with an explicit name from a given string."""
-    file = NamedTemporaryFile(suffix=suffix)
-    file_contents_bytes = str.encode(file_contents)
-    file.write(file_contents_bytes)
     file.seek(0)
-
-    return file
+    file_contents = file.read().encode()
+    return SimpleUploadedFile(file_name, file_contents)
```

Comment:
This change resolves some `Not a valid file` errors that were being raised in the serializers tests.